entrypoint.sh

```diff
@@ -77,7 +77,8 @@ function is_path_markdown {
 function c_mermaid {
   printf "Compiling: %s\n" "${1}"
   printf "Output to: %s\n" "${2}"
-  /node_modules/.bin/mmdc -p /mmdc/puppeteer-config.json -i "${1}" -o "${2}"
+  printf "Extra args: %s\n" "${MMDC_EXTRA_ARGS}"
+  /node_modules/.bin/mmdc -p /mmdc/puppeteer-config.json -i "${1}" -o "${2}" ${MMDC_EXTRA_ARGS}
```

Comment:
If this variable isn't passed, this will fail I think since we have `set -u` at the top. I don't have a standardized way of normalizing passed in variables, but maybe on line 20 add something like:

```bash
MMDC_EXTRA_ARGS="${MMDC_EXTRA_ARGS:-}"
```

Reply:
You're completely right.
I just tested without setting the `MMDC_EXTRA_ARGS` value, indeed it failed. I corrected it using your suggestion and now it's perfect without the variable set, as seen [here](https://github.com/sweet-delights/hybrid-platforms-conductor/runs/2438686523?check_suite_focus=true#step:6:9).

Thanks a lot for finding it!